packages/expect/src/__tests__/matchers.test.js

```diff
@@ -442,7 +442,9 @@ describe(
         jestExpect(big).toBeGreaterThanOrEqual(small);
       });
 
-      it(`{pass: false} expect(${small}).toBeGreaterThanOrEqual(${big})`, () => {
+      it(`{pass: false} expect(${small}).toBeGreaterThanOrEqual(${
+        big
```

Comment:
Ew...

Reply:
I thought `test`, `it` etc was special-cased to not break?

Reply:
Kind of - the special case is for arguments, not stuff inside the template literal.

Reply:
I can see the indentation is untouched, so probably not an issue